src/image/mod.rs

```diff
@@ -11,31 +11,38 @@
 //!
 //! # Examples
 //!
-//! ## Load a TGA image and draw it to a display
+//! ## Load an RGB565 raw data imge and display it
```

Comment:
I have only looked at the diff and didn't build the docs, but how often do we mention that other image formats can also be used via other crates? Maybe we should include this as a comment in this first example and point out that only the `raw` object is dependent on the image format and the code after that is common for all image formats.